pkg/database/vercode.go

```diff
@@ -83,15 +83,23 @@ func (v *VerificationCode) FormatSymptomDate() string {
 
 // IsCodeExpired checks to see if the actual code provides is the
 // short or long code and deteriminies if it is expired based on that.
-func (v *VerificationCode) IsCodeExpired(code string) bool {
+func (db *Database) IsCodeExpired(v *VerificationCode, code string) (bool, error) {
+	hmacedCode := code
+	if len(code) < 20 {
```

Comment:
Should we classify `20` with a name as a global? I am not really sure what `20` means. Is this < "short"?

Reply:
it was the previous max plaintext code length

can pull out to constant - or drop this and hmac the hmac value